concepticondata/concepticon.tsv

```diff
@@ -848,13 +848,13 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 847	THIRSTY	Food and drink	Feeling the need to drink.	Property	
 848	THIRST	Food and drink	A physiological need to drink.	Person/Thing	
 849	SPRING (OF WATER)	The physical world	A place where ground water flows naturally from a rock or the soil onto the land surface or into a body of surface water.	Person/Thing	
-850	SQUARE	Spatial relations	An open area in a town, sometimes including the surrounding buildings.	Person/Thing	
+850	SQUARE	Spatial relations	A geometrical figure with four sides that meet each other on right angles, or something resembling its shape.	Person/Thing	
 851	STABLE	Animals	A building or structure usually with stalls that is used to house and feed horses, cattle or other animals.	Person/Thing	
 852	PRIEST	Religion and belief	A minister of the Catholic church empowered to administer the sacraments, most particularly that of the Eucharist or Holy Communion, as well as those of confession and extreme unction.	Person/Thing	
 853	ALTAR	Religion and belief	Table for religious ceremonies.	Person/Thing	
 854	KETTLE	Food and drink	A metal container used to boil any kind of liquid.	Person/Thing	
 855	SEIZE	Possession	To take possession of by force or authority.	Action/Process	
-856	HIRE	Possession	To hold under a lease or rental agreement of goods and services.	Action/Process	
+856	HIRE	Possession	To employ a person.	Action/Process	
```

Comment:
To employ a person or people in exchange of some kind of remuneration, usually a payment.